lightcurve/views.py

```diff
@@ -175,3 +176,16 @@ def all_lightcurve(request):
 
     return render_to_response('lightcurve_list.html', {"contexts": context_list, "request_all": True},
             context_instance=RequestContext(request))
+
+
+def download_file(request, lightcurve_id):
+    file_type = request.GET.get('file_type')
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="LightCurve' + lightcurve_id + '.' + file_type + '"'
+
+    if file_type == "csv":
+        writer = csv.writer(response)
+        writer.writerow(['First row', 'Foo', 'Bar', 'Baz'])
+        writer.writerow(['Second row', 'A', 'B', 'C', '"Testing"', "Here's a quote"])
```

Comment:
Looking good!

Take a look at plot_lightcurve_json to see how x, y data is built for graphs.  This will be similar to what we need to do for CSV: https://github.com/typpo/astrokit/blob/master/lightcurve/views.py#L39

It might wind up looking something like:

```
writer.writerow(['Datetime', 'JD', 'Mag instrumental', 'Mag standard', 'Mag std'])
for analysis in analyses:
   writer.writerow([analysis.image_datetime, get_jd_for_analysis(analysis), result['mag_instrumental'], result['mag_standard'], result['mag_std'])
```